RZAssemblage/Private/RZFRCAssemblage.m

```diff
@@ -0,0 +1,195 @@
+//
+//  RZFRCAssemblage.m
+//  RZAssemblage
+//
+//  Created by Brian King on 1/29/15.
+//  Copyright (c) 2015 Raizlabs. All rights reserved.
+//
+
+#import "RZFRCAssemblage.h"
+#import "NSIndexPath+RZAssemblage.h"
+#import "RZArrayAssemblage.h"
+#import "RZAssemblageDefines.h"
+#import "RZAssemblageChangeSet+Private.h"
+
+
+#define RZAssertIndexPathLength(indexPath, offset) RZRaize(indexPath.length <= ((self.hasSections ? 2 : 1) + offset), @"Index Path %@ has length of %lu, expected index <= %d", indexPath, (unsigned long)indexPath.length, ((self.hasSections ? 2 : 1) + offset))
+#define RZAssertContainerIndexPath(indexPath) RZAssertIndexPathLength(indexPath, -1)
+#define RZAssertItemIndexPath(indexPath) RZAssertIndexPathLength(indexPath, 0)
+
+@interface RZFRCAssemblage() <NSFetchedResultsControllerDelegate>
+
+@property (assign, nonatomic, readonly) BOOL hasSections;
+@property (strong, nonatomic) RZAssemblageChangeSet *changeSet;
+
+@end
+
+@implementation RZFRCAssemblage
+
+@synthesize delegate = _delegate;
```

Comment:
Most or all of these `@synthesize` are no longer needed since `RZAssemblage` is no longer a protocol